tests/e2e/etcd_spawn_nocov.go

```diff
@@ -38,3 +43,23 @@ func spawnCmd(args []string) (*expect.ExpectProcess, error) {
 	}
 	return expect.NewExpectWithEnv(args[0], args[1:], env)
 }
+
+func mergeEnvVariables(envVars map[string]string) []string {
```

Comment:
Is it from main branch?

Reply:
YES.  

There is no function `SpawnCmdWithLogger` in `releasae-3.4`, and a new function `spawnCmdWithEnv` is added in this PR. It is a minor difference from the original PR. It should be OK.